twistlock/datadog_checks/twistlock/config.py

```diff
@@ -13,9 +11,7 @@ def __init__(self, instance):
         if self.url.endswith('/'):
             self.url = self.url[:-1]
 
-        self.username = instance.get('username')
-        self.password = instance.get('password')
-
         self.tags = instance.get('tags', [])
 
-        self.ssl_verify = is_affirmative(instance.get('ssl_verify', True))
+        self.username = instance.get('username')
```

Comment:
Can username and password be removed from the config? (Probably this means editing tests)

Reply:
Seems the credentials are required for `twistlock`, that's why username and password are collected in config: https://github.com/DataDog/integrations-core/blob/master/twistlock/datadog_checks/twistlock/twistlock.py#L56-L57

Reply:
Will Twistlock itself raise an exception? because if it does that check is pretty useless

Reply:
Yes, agree, the value is not high since, as you said, Twistlock is probably going to raise an error when use the API if the username/password are wrong or missing. However, that check will make it fail earlier, since credentials are required, which is good (not high value, but still a good thing IMHO).